mm2src/mm2_main/tests/integration_tests_common/mod.rs

```diff
@@ -92,6 +93,71 @@ pub async fn enable_z_coin_light(
         }
     }
 }
+pub async fn enable_z_coin_light_with_changing_height(
```

Comment:
I don't think we need this new test helper function, we can reuse `enable_z_coin_light` by adding `starting_date: Option<u64>,` to it